### PR TITLE
OpenRouter Integration

### PR DESCRIPTION
**Pull request by Anindyadeep**
This PR:

- Solves issue #20

### CODE REVIEW

**Comment by Anindyadeep** [comment]
I will merge this after PR #47 get's merged and also don't I need to change the slug here? I mean there are still some identical

File: prem_utils/models.json
Comment (filopedraz) [inline prem_utils/models.json:691]: No slug conflicts here?

**Review by filopedraz — CHANGES_REQUESTED** [review]

File: prem_utils/connectors/openrouter.py
Comment (filopedraz) [inline prem_utils/connectors/openrouter.py:28]: Now that you put `openrouter/` in front of each element, you don't need this if/else logic. 

```python
model = model.replace("openrouter/", "", 1)
```

Will already replace only the first occurence.

**Review by filopedraz — CHANGES_REQUESTED** [review]

File: prem_utils/connectors/openrouter.py
Comment (Anindyadeep) [reply prem_utils/connectors/openrouter.py]: Done

**Review by filopedraz — APPROVED** [review]

**Pull request merged**
